src/createServer.js

```diff
@@ -1,3 +1,44 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable max-len */
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { errorValidation } = require('./errorValidation');
+
+const createServer = () => {
+  const server = http.createServer((request, response) => {
+    response.setHeader('Content-Type', 'application/json');
+
+    const [pathName, queryParams] = request.url.split('?');
+    const text = pathName.slice(1);
+    const params = new URLSearchParams(queryParams);
+    const toCase = params.get('toCase');
+
+    const errors = errorValidation(text, toCase);
+
+    if (errors.length) {
+      const errorResponse = {
+        errors,
+      };
+
+      response.statusCode = 400;
+      response.statusText = 'Bad request';
+
+      response.end(JSON.stringify(errorResponse));
```

Comment:
Since you send responses in 2 places you can create a separate function `sendResponse(res, data, statusCode)` and move this logic there